packages/repack/android/src/main/java/com/callstack/repack/CodeSigningUtils.kt

```diff
@@ -104,5 +105,27 @@ class CodeSigningUtils {
                 throw Exception("The bundle verification failed because the bundle hash is invalid.")
             }
         }
+
+        fun extractBundleAndToken(fileContent: ByteArray): Pair<ByteArray, String?> {
+            val signatureSize = 1280
+            val startingSequence = "/* RCSSB */"
```

Comment:
Please, for the sake of future generations, add a comment what the RCSSB stands for :)